frontend/src/api/prometheus/usePrometheusQueryRange.ts

```diff
@@ -62,10 +64,4 @@ const usePrometheusQueryRange = <T = PrometheusQueryRangeResultValue>(
   return [...useFetchState<T[]>(fetchData, []), pendingRef.current];
 };
 
-export const defaultResponsePredicate: ResponsePredicate = (data) => data.result?.[0]?.values || [];
-
-export const prometheusQueryRangeResponsePredicate: ResponsePredicate<
-  PrometheusQueryRangeResponseDataResult
-> = (data) => data.result || [];
```

Comment:
You have removed two exports being used by other files.
`npm run test:type-check` fails.

Reply:
I apologies for this. I missed to revert back after changing `usePrometheusQueryRange.ts` to debugging. reverted it back. Thanks.